### DISCUSSION TITLE
[Feature] Add more statistics

### DISCUSSION THREAD

**Discussion by edbourque0** [johanohly/AirTrail#151] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add more stats to the statistics page. Here are the ones I would like to see : 
- Number of airlines
- Most used airline
- Number and names of airports visited

Thank you!

**Comment by johanohly** [comment]
Hey, thank you very much for this feature request. More statistics are definitely on my list, I'll keep your ideas in mind!

**Comment by johanohly** [comment]
This has been partially added in v2.2.1! Do let me know if you would still like the missing statistics as well.

**Reply by edbourque0** [reply]
Thanks, the stats are great!
